serverless/Dockerfile.j2

```diff
@@ -13,6 +13,9 @@ RUN echo "Starting ..." && \
     echo "Install basics Python tools" && \
     pip install --upgrade pipenv==${PIPENV_VERSION} && \
     echo "Done basic python tools install" && \
+    echo "Install aws-cli" && \
+    pip install awscli && \
```

Comment:
Could you please add a test in config.yml?

Reply:
BTW the maintainer is missing, please add him :)

Reply:
done both test and addition of the maintainer